### ISSUE TITLE
Strings.PlatformNotSupported_DataSqlClient After Upgrade from Ubuntu 20.04 LTS to 22.04 LTS

### ISSUE THREAD

**Issue by gwduvalljr** [dotnet/efcore#28788]
My environment:
Dell XPS 13 9310
Several net6.0 projects (web, API, common framework.)
Microsoft.Data.SqlClient 5.0.0
All get deployed to Azure. 

VSCode
- Version: 1.70.2
- Commit: e4503b30fc78200f846c62cf8091b76ff5547662
- Date: 2022-08-16T05:36:25.715Z
- Electron: 18.3.5
- Chromium: 100.0.4896.160
- Node.js: 16.13.2
- V8: 10.0.139.17-electron.0
- OS: Linux x64 5.14.0-1048-oem snap

Everything was fine. I decided to upgrade my Dell XPS13 from Ubuntu 20.04 LTS to 22.04 LTS when prompted by the Software Updater. 

The upgrade went OK (not perfect) and I ended up removing and reinstalling .NET6 SDK, but after an hour or so I was able to build the projects with no errors or issues and thought I was back to being OK. Then I tried to run them. On the first access to the database, I received the Strings.PlatformNotSupported_DataSqlClient exception. 

So, is Ubuntu 22.04 LTS not supported or is there some other thing I need to do to "clean up" from the upgrade? At this point, I have moved to a different computer to work until I can get this fixed. Any ideas?

**Comment by roji** [comment]
Likely a duplicate of #28786 and/or https://github.com/dotnet/SqlClient/issues/1427, and in any case most likely a SqlClient issue.

**Comment by gwduvalljr** [comment]
Wiped my laptop and installed everything clean. All is working now. 
Ubuntu: Ubuntu 22.04.1 LTS
Gnome: 42.2
VSCode: 1.70.2
.net: 6.0.108/sdk
Microsoft.Data.SqlClient: 5.0.0

**Comment by roji** [comment]
Duplicate of https://github.com/dotnet/SqlClient/issues/1427

**Comment by luanmm** [comment]
Some people disagree that is [something wrong with SqlClient](https://github.com/dotnet/SqlClient/issues/1423#issuecomment-1231821423). I can confirm that is not closed (at least in EF Core 6.0.8).

**Comment by roji** [comment]
@luanmm see the subsequent discussion in https://github.com/dotnet/SqlClient/issues/1423 - there seem to be multiple issues being discussed there, and none of them seem related to EF in any way.

**Comment by luanmm** [comment]
> @luanmm see the subsequent discussion in [dotnet/SqlClient#1423](https://github.com/dotnet/SqlClient/issues/1423) - there seem to be multiple issues being discussed there, and none of them seem related to EF in any way.

Yep. Digging deeper, I think is more related on how libraries are loaded based on the identified runtime. I reported, referencing all the issues that I've found related to this problem, [in the runtime repo](https://github.com/dotnet/runtime/issues/75427). Hope that something could be answered there regarding this problem.

Thanks for all the point of views and tips given until now.
